jupyterlab_gcedetails/jupyterlab_gcedetails/handlers.py

```diff
@@ -152,3 +164,16 @@ async def get(self):
     self.details[UTILIZATION] = get_resource_utilization()
     self.details[GPU] = await get_gpu_details()
     self.finish(self.details)
+
+class PricelistHandler(APIHandler):
+  """Returns pricing information for instance price estimation."""
+
+  async def get(self):
+    try:
+      pricelist = await get_pricelist()
+      response = pricelist.get('gcp_price_list', {})
```

Comment:
I would probably extract the `gcp_price_list` key from the returned JSON object in the get_pricelist function just to keep the logic in the handler as simple as possible and bury the implementation specifics for the endpoint in the accessor function.